### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'android.graphics.drawable.Drawable va.c.a()' on a null object reference

### ISSUE THREAD

**Issue by sarankumar-ns** [zoho/salesiq-mobilisten-flutter#18]
I was getting NullPointerException.

````
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'android.graphics.drawable.Drawable va.c.a()' on a null object reference
       at com.zoho.livechat.android.operation.SalesIQApplicationManager$8$1.run(SalesIQApplicationManager.java)
       at android.os.Handler.handleCallback(Handler.java:751)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:186)
       at android.app.ActivityThread.main(ActivityThread.java:6513)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:914)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:804)
````

Another thing is if I use any functions in salesIQ like `init`, `registerVisitor`,  `unregisterVisitor`, then the Chat widget automatically becomes visible. So, after every operation, I need to set `showLauncher` to false.

Also, the above exception happened in 
 * After the user Sign In, I registered the visitor in SalesIQ and navigate to the next page.

**Comment by DavidBhahirathan** [comment]
Hello @sarankumar-ns  ,

Sorry for the inconvenience. We'll check and fix both issues and keep you updated here. 

Regards,
Bhahirathan M.

**Comment by DavidBhahirathan** [comment]
Hi @sarankumar-ns ,

We have addressed this issue in version [2.0.0](https://github.com/zoho/SalesIQ-Mobilisten-flutter/releases/tag/v2.0.0) with a fix that will resolve this crash. If the crash persists, please send us the exact crash with the file name and its line number.

For further queries, please feel free to write us at [support@zohosalesiq.com](mailto:support@zohosalesiq.com). We will be happy to assist you.

Regards,
Bhahirathan M,
Zoho SalesIQ.
